include/openbsd-blowfish.h

```diff
@@ -41,6 +41,22 @@
 #define BLF_MAXKEYLEN ((BLF_N-2)*4)	/* 448 bits */
 #define BLF_MAXUTILIZED ((BLF_N+2)*4)	/* 576 bits */
 
+#ifdef	SOLARIS2
```

Comment:
I _really_ try to avoid platform-specific `#ifdef`s like these; they are hard to maintain.  Better is to just detect when unexpected `typedef`s are used, and act accordingly.